contracts/market/CreateLoanWithNFTFacet.sol

```diff
@@ -115,6 +115,23 @@ contract CreateLoanWithNFTFacet is ReentryMods, PausableMods {
                     version,
                     tokenData_
                 );
+        } else if (version == 3) {
+            // get loan size from first function
+            uint256 loanSize1 = CreateLoanWithNFTFacet._takeOutLoanProcessNFTs(
```

Comment:
if this is for allowed loan size, isn't it better to name them `allowedLoanSize1`, `allowedLoanSize2`?